zerver/views/auth.py

```diff
@@ -305,11 +320,11 @@ def finish_desktop_flow(request: HttpRequest, user_profile: UserProfile, otp: st
     iv = secrets.token_bytes(12)
     desktop_data = (iv + AESGCM(key).encrypt(iv, token.encode(), b"")).hex()
     context = {
-        "desktop_data": desktop_data,
+        "redirect_data": desktop_data,
```

Comment:
Just a note that before merging this we should give some thought to how we want to name this - `redirect_data` seems too vague for me - though I don't think it's worth overthinking the name right name, so we can do that at the end